### PR TITLE
Implement external "expected errors" configuration by plugin

### PR DESCRIPTION
**Pull request by cbush**
- Adds yargs handling to parse args (but shouldn't impact existing users)
- Adds flag for config.js
- Moves expectedErrors to "defaultConfig.js"
- Imports expectedErrors from provided config.js, falling back on defaultConfig.js if option not provided

### CODE REVIEW

File: defaultConfig.js [hunk 1]
```diff
@@ -0,0 +1,8 @@
+module.exports = {
+  expectedErrors: [
+    /(WARNING|ERROR)\(sdk\/java\/api.*/,
+    /ERROR #98124  WEBPACK/,
+    /WARNING.*: Directive "container" has been deprecated/,
+    /Title (overline|underline) too (short|long)/, // Seriously?!
```
Comment (mongodben) [inline defaultConfig.js]:
```suggestion
    /Title (overline|underline) too (short|long)/, 
```

i know but..

File: src/index.ts [hunk 2]
```diff
@@ -161,32 +199,33 @@ ${reason}`,
...
+      return yargs
+
+        .positional("actorOwnerRepoBranch", {
+          describe: "actor/owner/repo/branch to check build",
+          type: "string",
+        })
+        .option("config", {
+          string: true,
+          description:
+            "Path to configuration js. See defaultConfig.js for an example.",
```
Comment (mongodben) [inline src/index.ts]:
```suggestion
            "Path to configuration JavaScript file. See defaultConfig.js for an example.",
```

**Review by mongodben — APPROVED** [review]
lgtm, few small things

File: defaultConfig.js (hunk 1 above)
Comment (cbush) [reply defaultConfig.js]: Inherited from original

**Pull request merged**